src/App.js

```diff
@@ -1,19 +1,19 @@
 import './App.css';
-import React from 'react';
-import { Routes, Route } from 'react-router-dom';
-import BookList from './components/BookList';
+import { Provider } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
 import NavBar from './components/NavBar';
```

Comment:
- Hey, please, kindly update the import statement here as it should be navbar not navBar to fix the error thrown in the browser in running your app below:

`import NavBar from './components/Navbar';`